packages/components/src/placeholder/style.scss

```diff
@@ -38,15 +39,14 @@
 }
 
 .components-placeholder__label {
-	display: flex;
 	font-weight: 600;
-	margin-bottom: $grid-unit-20;
 	align-items: center;
+	display: inline-flex;
```

Comment:
What is the reason for changing from `flex` to `inline-flex`?

Reply:
That was an older iteration. Good catch, changed it back.